src/DotNetEnv/Parser.cs

```diff
@@ -79,8 +79,10 @@ private static string RemoveExportKeyword(string line)
 
         private static bool IsQuoted(string s)
         {
-            return (s[0] == '"' && s[s.Length - 1] == '"')
-                || (s[0] == '\'' && s[s.Length - 1] == '\'');
+            return s.Length > 1 && (
```

Comment:
handling just EMPTY requires this be `s.Length > 0` but that revealed a case of a single quote breaking the logic as well, so then it must be `s.Length > 1`